etc/kayobe/kolla/config/neutron/dhcp_agent.ini

```diff
@@ -0,0 +1,4 @@
+dhcp_broadcast_reply = True
+interface_driver = multi
```

Comment:
I wonder if these should be in the `[DEFAULT] section? The interface_driver seems to be there in the default configuration file [1]. If this is the case and they are being ignored, do we need them at all?  


[1]